packages/adblocker-webextension/src/index.ts

```diff
@@ -128,6 +131,43 @@ export function updateResponseHeadersWithCSP(
   return { responseHeaders };
 }
 
+const readableMimeTypes = new Set([
```

Comment:
please rename to be full capitalised

Reply:
Fixed in [`658f616` (#14)](https://github.com/seia-soto/adblocker/pull/14/commits/658f6168de9d534d72b3b1db2e56a43fef716373)